.github/workflows/swift-toolchain.yml

```diff
@@ -928,21 +930,12 @@ jobs:
         if: inputs.build_os == 'Darwin'
 
       - name: Install Swift Toolchain
-        if: inputs.build_os == 'Windows' && inputs.build_arch == 'amd64'
-        uses: compnerd/gha-setup-swift@b6c5fc1ed5b5439ada8e7661985acb09ad8c3ba2 # main
+        uses: compnerd/gha-setup-swift@6c9f2db7c3155c57fe35f160bcd5cf5859b9c1ba # main
```

Comment:
Please keep all `compnerd/gha-setup-swift` hashes in sync.

Reply:
Done